x/auth/client/context/context.go

```diff
@@ -25,17 +25,8 @@ type TxContext struct {
 // NewTxContextFromCLI returns a new initialized TxContext with parameters from
 // the command line using Viper.
 func NewTxContextFromCLI() TxContext {
-	// if chain ID is not specified manually, read default chain ID
```

Comment:
Why did we remove this? What if I don't have a config, I still want this to be auto-populated since I'm lazy.